nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -712,6 +712,25 @@ auth required pam_succeed_if.so uid >= 1000 quiet
      For further reference, please read <link xlink:href="https://github.com/NixOS/nixpkgs/pull/68953">#68953</link> or the corresponding <link xlink:href="https://discourse.nixos.org/t/predictable-network-interface-names-in-initrd/4055">discourse thread</link>.
     </para>
    </listitem>
+   <!-- TODO re-add changelog
```

Comment:
Note to self: as soon as we have enough approvals here I should update the release notes.

Reply:
Release-notes were updated during the latest push.